src/assets/scss/main.scss

```diff
@@ -44,6 +64,7 @@ $fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
 }
 
 .item-select {
+  /* Quality text with white background */
```

Comment:
?

Reply:
These quality classes are used for the same purpose of the q* classes above but only where the background of the text is white.
I improved the comment.